arcade/types.py

```diff
@@ -17,7 +17,10 @@
     Sequence,
     Tuple,
     Union,
-    TYPE_CHECKING, TypeVar
+    TYPE_CHECKING,
+    TypeVar,
+    TypeAlias,
+    Literal,
 )
 from typing_extensions import Self
```

Comment:
```suggestion
    Literal,
)
from typing_extensions import Self, TypeAlias
```
We have to import `TypeAlias` from the backports package since it's a 3.10 feature.